cob_bringup/components/cob4_base.launch

```diff
@@ -19,11 +20,18 @@
 		<arg name="use_external_sync" value="$(arg use_external_sync)"/>
 		<arg name="start_external_sync" value="$(arg start_external_sync)"/>
 	</include>
-	<node if="$(arg sim)" ns="$(arg component_name)" pkg="cob_helper_tools" type="fake_driver.py" name="fake_driver" cwd="node" respawn="false" output="screen"/>
+	<rosparam unless="$(arg sim)" ns="$(arg component_name)/driver" command="load" file="$(arg pkg_hardware_config)/common/cob4_base_driver.yaml"/>
 
-	<include file="$(find cob_bringup)/controllers/base_controller_plugin.launch">
-		<arg name="robot" value="$(arg robot)"/>
+	<include unless="$(arg sim)" file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
```

Comment:
please consider https://github.com/ipa320/cob_robots/pull/618#discussion_r100999649 and add a launch file for the stuck_detector to cob_bringup